doc/src/doc-updates/sui-breaking-changes.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Sui Breaking Changes in Release .28
+---
+
+The next release of Sui, release 0.28, includes the breaking changes described in this topic. A breaking change is one that introduces new, or changed, Sui functionality that causes existing apps and implementations to stop functioning as expected. Updates to this topic for next week include information about how to resolve or update your project that might break as a result of these changes.
+
+**[Major breaking change]** - The new Programmable Transactions feature introduces a new type of transaction that replaces both batch transactions and normal transactions (with the exception of special system transactions). These transactions allow for a series of Commands (mini transactions of sorts) to be executed, where the results of commands can be used in following commands. For more information, see the [Programmable Transactions RFC](https://forums.sui.io/t/rfc-planned-feature-programmable-transactions/3823).
+
+**[Major breaking change]** - `SuiAddress` and `ObjectID` are now 32 bytes long instead of 20 bytes (in hex, the `len` increases from 40 to 64). If your software interacts with any `ObjectID` and `SuiAddress`, you must update it to use updated addresses and redeploy it. [PR 8542](https://github.com/MystenLabs/sui/pull/8542)
+
+**[Breaking change]** - Changes to Gas Budget to use SUI rather than gas units. This removes the concept of gas units from any user-related API operations. This does not change the format of `TransactionData` (u64). This is not a breaking change in the sense that the current format no longer works, but rather requires you to reconsider how you use gas budgets.
+
+**[Breaking change]** - Currently, transactions require a single coin to pay for gas. This sometimes results in users needing to make separate transactions (such as `PaySui`) to merge coins before completing a transaction, which can also increase the cost of the transaction.
+
```

Comment:
Remove linespace ?

Reply:
the whitespace makes it easier to read
<img width="413" alt="image" src="https://user-images.githubusercontent.com/109545725/222602299-c693b616-e955-4500-be05-0ef40e401161.png">

Reply:
could do bullets instead
<img width="1274" alt="image" src="https://user-images.githubusercontent.com/109545725/222603682-9acf96e4-6385-4b2c-949a-4eba6fe1352b.png">

Reply:
Even though the following section is "part" of the previous one on this change, I think the space is still preferred for readability. IMO, the bolding acts as bullets but don't think they're necessary. Wouldn't be mad if they're added, either, tho.